app/Console/Command/ProcessDuplicatePlaces.php

```diff
@@ -173,6 +175,11 @@ public function execute(InputInterface $input, OutputInterface $output): int
 
     private function askConfirmation(InputInterface $input, OutputInterface $output, int $count): bool
     {
+        if ($input->getOption(self::FORCE) === true) {
+            $output->writeln("This action will process {$count} clusters");
```

Comment:
I do not think this feedback is very useful in this case, `force` options should only be run via automated `cronjob` or via `Jenkins`. Besides, it cannot be stopped. We also do not give this type of feedback on other `force` commands.